core/src/player.rs

```diff
@@ -1932,6 +1923,8 @@ pub struct PlayerBuilder {
 
     // Misc. player configuration
     autoplay: bool,
+    scale: StageScaleMode,
```

Comment:
IIUC `scale` and `force_scale` fields can be merged into a single `forced_scale_mode: Option<StageScaleMode>` field, where `None` means no scale mode is forced and `Some(...)` means, well, some scale mode is forced.
I see it This is more idiomatic and it prevents the situation where the value of `scale` is meaningless when `force_scale` is `false`.

Reply:
`scale` sets the scale mode that the player starts with. ActionScript can change it at any time. `forceScale` prevents ActionScript from changing it. Flash Player never allowed forcing the scale mode, so whatever you set could be changed by the movie's ActionScript.